seatunnel-connectors-v2/connector-amazonsqs/src/main/java/org/apache/seatunnel/connectors/seatunnel/amazonsqs/source/AmazonSqsSource.java

```diff
@@ -113,7 +113,8 @@ private void setDeserialization(Config config) {
             MessageFormat format = ReadonlyConfig.fromConfig(config).get(FORMAT);
             switch (format) {
                 case JSON:
-                    deserializationSchema = new JsonDeserializationSchema(false, false, typeInfo);
+                    deserializationSchema =
+                            new JsonDeserializationSchema(false, false, typeInfo, null);
```

Comment:
```suggestion
                            new JsonDeserializationSchema(false, false, typeInfo);
```

suggestion update all `xxxDeserializationSchema.java`

```java
class JsonDeserializationSchema {
    public JsonDeserializationSchema(agrs1, args2, args3){
        this(agrs1, args2, args3, null);
    }

    public JsonDeserializationSchema(agrs1, args2, args3, args4){
        ...
    }
}
```

Reply:
Done ，thanks